build_sysext

```diff
@@ -184,15 +186,15 @@ mkdir "${BUILD_DIR}/install-root"
 mkdir "${BUILD_DIR}/workdir"
 mount -t overlay overlay -o lowerdir="${BUILD_DIR}/fs-root${pkginfo_lowerdirs}",upperdir="${BUILD_DIR}/install-root",workdir="${BUILD_DIR}/workdir" "${BUILD_DIR}/install-root"
 
-REPO_BUILD_ID="$(source "${REPO_MANIFESTS_DIR}/version.txt"; echo "$FLATCAR_BUILD_ID")"
-REPO_FLATCAR_VERSION="$(source "${REPO_MANIFESTS_DIR}/version.txt"; echo "$FLATCAR_VERSION")"
+REPO_BUILD_ID=$(source "${REPO_MANIFESTS_DIR}/version.txt"; echo "$FLATCAR_BUILD_ID")
+REPO_FLATCAR_VERSION=$(source "${REPO_MANIFESTS_DIR}/version.txt"; echo "$FLATCAR_VERSION")
 VERSION_BOARD=$(source "${BUILD_DIR}/fs-root/usr/lib/os-release" && echo "$VERSION")
 
-if [ -z "$REPO_BUILD_ID" ] ; then
+if [[ -z $REPO_BUILD_ID ]] && [[ ! -f $BOARD_ROOT/etc/portage/package.use/official ]]; then
```

Comment:
Can this rely on `${COREOS_OFFICIAL}` instead?

Reply:
it's only set from the `ci-automation`, so a user rebuilding a release tag will not have it set unless they want to produce an "official" image.